### DISCUSSION TITLE
Write EvResult for Evs arriving but not directly start charging?

### DISCUSSION THREAD

**Discussion by danielfeismann** [ie3-institute/simona#792] [General]
(Hopefully only) a small point for discussion, we generally only write results when there is a change. In the case of the Evs, we currently write a result with 0 kW when the vehicle arrives and does not yet start charging directly. I have a tendency not to write an entry in that case. With regard to flexibility, you can of course also say that from that point onwards it is possible to charge/discharge and we want to record this event. On the other hand, there is also the option of writing out the FlexResults. Do you have any additional opinions / comments?

**Comment by t-ober** [comment]
I'm fine with not writing the 0 kW result. Does not sound super useful to me other than maybe debugging to see when the ev arrived at the charging station.

**Comment by sebastian-peter** [comment]
I think it's not necessary as long as we always write out 0 kW whenever an EV stops charging or departs

**Comment by sebastian-peter** [comment]
Guess this can be closed, as we reached agreement on keeping the status quo

**Reply by danielfeismann** [reply]
I agree on closing this. However, I don't see that we agree to keep the status quo as I would prefer to remove the result event with 0 kW when arriving. Thus, we should transfer this into an issue at SIMONA.

**Reply by sebastian-peter** [reply]
Sorry, then I misunderstood the outcomes of our discussion.

**Reply by danielfeismann** [reply]
Well, the idea would be only writing results when the charging changes:
arrival but don't charge at the moment -> no result entry
arrival and directly charging -> write result entry
stop charging and departs -> write result entry
stop charging and not departing -> write result entry
already stopped charging and depart -> no result entry

**Reply by sebastian-peter** [reply]
Ok, then I indeed understood properly, but just got confused. :D
